components/SortByChronologic/index.js

```diff
@@ -53,20 +53,15 @@ export const MoviesByChronologic = ({
   }
 
   return (
-    <MovieCardsList
-      as={motion.div}
-      initial={{ opacity: 0, y: -50 }}
-      animate={{ opacity: 1, y: 0 }}
-      exit={{ opacity: 0, y: -50 }}
-    >
+    <MovieCardsList>
       {sortedMovies.map((movie, index) => (
         <StyledMovieCard
           key={movie.id}
           as={motion.div}
           initial={{ opacity: 0, y: -50 }}
           animate={{ opacity: 1, y: 0 }}
           exit={{ opacity: 0, y: -50 }}
-          transition={{ delay: index * 0.2 }}
+          transition={{ delay: index * 0.0 }}
```

Comment:
What's the point in setting the delay to 0.0? As you are multiplying it with index, it will always result in 0... so I guess you could completely remove the transition prop? 🤔